tests/core/_device/test_device_save_loader.py

```diff
@@ -142,3 +148,48 @@ def save_my_device():
         assert np.array_equal(
             yaml_content[1]["child2.sig1"]["VAL2"], np.array([1, 1, 1, 1, 1])
         )
+
+
+async def test_yaml_formatting_no_phase(device_with_phases, tmp_path):
+    RE = RunEngine()
+    await device_with_phases.child1.sig1.set("test_string")
+    table_pv = {"VAL1": np.array([1, 2, 3, 4, 5]), "VAL2": np.array([6, 7, 8, 9, 10])}
+    await device_with_phases.child2.sig1.set(table_pv)
+
+    # Create save plan from utility functions
+    def save_my_device():
+        signalRWs = walk_rw_signals(device_with_phases)
+        values = yield from get_signal_values(signalRWs)
+        phases = device_with_phases.sort_signal_by_phase(device_with_phases, values)
+        save_to_yaml(phases, path.join(tmp_path, "test_file.yaml"))
+
+    RE(save_my_device())
+
+    with open(path.join(tmp_path, "test_file.yaml"), "r") as file:
+        assert file.read() == """\
+- {child1.sig1: test_string}
+- child2.sig1:
+    VAL1: [1, 2, 3, 4, 5]
+    VAL2: [6, 7, 8, 9, 10]
+"""
```

Comment:
Black isn't happy, how about
```suggestion
        expected = """\
- {child1.sig1: test_string}
- child2.sig1:
    VAL1: [1, 2, 3, 4, 5]
    VAL2: [6, 7, 8, 9, 10]
"""
        assert file.read() == expected
```